Native Android Client

armanrahman22 opened the pull request:
Hey y'all! Here is the native Android client. I'm still working on testing, but everything else should (hopefully) work. Lemme know if I need to change anything!

bengreenier commented:
hey @armanrahman22 thanks so much for this! I can't wait to take a deeper look! 💯:rocket: 

A couple quick things I did notice right away though:

+ Do we need the inner `3DToolkitAndroid` directory? if not imo it would be cleaner to just have everything at `Android-Native`.
+ Can you write a brief README in `Android-Native` describing at least the dev environment/how to build this code?

armanrahman22 commented:
Hey y'all @bengreenier  @anderm thanks for the comments! I just pushed changes to the controls and I moved the directory and added a quick readme. Lemme know if I need anything else!

anderm commented:
See #130 #131 for remaining issues
